### DISCUSSION TITLE
Release 1.18

### DISCUSSION THREAD

**Discussion by webspiderteam** [webspiderteam/Bluetooth-DMM-For-Windows#23] [Announcements]
Added Base Value and Symbol to MQTT

<hr /><em>This discussion was created from the release <a href='https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/releases/tag/WPFRelease1.18'>Release 1.18</a>.</em>

**Comment by ludwich66** [comment]
Hello engine,
sorry, i would like to change "Added Base Value and Symbol to MQTT" to "Added Base-Value and Base-Symbol to MQTT" in release page. "Symbol" is my opinion an icon or picture rather than a unit-(symbol).

And thanks for the new release!

**Reply by webspiderteam** [reply]
Please its not engine. just Engin. its not even relevant with engine. 
ofcourse I couldnt name it until now. thanks

**Reply by ludwich66** [reply]
Oh sorry a typo.
I know that almost all Turkish first names translate to something "beautiful". mine also has a history that means something like "the farmer".

L ;-)

**Comment by ludwich66** [comment]
Engin, 

ist Line "Sample Output" overlapping on your screen also?

![image](https://user-images.githubusercontent.com/12202733/184414547-344a9777-0369-4c30-93b4-c16e1a036704.png)

**Reply by webspiderteam** [reply]
Yes. O didn't thought it can be that much long

**Reply by webspiderteam** [reply]
Fixed

**Comment by ludwich66** [comment]
Hi Engin,
first Step to new Wiki is done.
Please check and give feedback.
I have added a note in Readme.md to jump to Wiki

ludwich

**Reply by webspiderteam** [reply]
Ohh. That looks nice :) Thank you very much :) I released new one with some fixes. I think its done all posible things are made. I dont have any improvements right now.
